src/flask_pyoidc/redirect_uri_config.py

```diff
@@ -60,4 +60,3 @@ def _parse_legacy_config(config):
         full_uri = scheme + '://' + redirect_domain + '/' + endpoint
 
         return full_uri, endpoint
-
```

Comment:
Unrelated change.

Reply:
PEP8 compliance: Leave a line at the end of the file.

Reply:
Yes, great catch!
But since no other changes were made in this file in this PR it's not really related to this.
Minor formatting fixes like this would be ok for already changed files, but for bigger/other fixes a separate PR would be better (this might not be the only file in need of some cleaning 🙂 ).